### ISSUE TITLE
Can i share scenes like a dashboard panel?

### ISSUE THREAD

**Issue by Fi1osof** [grafana/scenes#853]
Hello!

I can share dashboard panel 
![image](https://github.com/user-attachments/assets/71ce1340-e460-44ee-9227-484a40c570b8)
![image](https://github.com/user-attachments/assets/0b2e1082-cd5e-43ec-9e49-5e353ce94c78)

But can i share scenes? 
![image](https://github.com/user-attachments/assets/ad7f9afb-7a45-4a47-838a-7b6e995d7e7e)

Thanks!

**Comment by torkelo** [comment]
no, not currently (and no plans), it's something up to each scene app dev to develop in their app.

If there is an easy way to do it generically it might make sense to put it into the lib

**Comment by Fi1osof** [comment]
@torkelo, thanks for answer!

Before i try to render in my site grafana panels directly, using api, @grafana/scenes and @grafana/ui, but i can't :-( Looking for grafana/grafana project, a see, it's used [public/app](https://github.com/grafana/grafana/tree/main/public/app) directory and no way to import views from there. @grafana/ui also no have all widgets.
